packages/11ty/_includes/components/head.js

```diff
@@ -57,6 +57,8 @@ module.exports = function(eleventyConfig, globalData) {
         <link rel="version-history" href="${publication.repositoryUrl}">
 
         <script src="https://cdn.jsdelivr.net/npm/@digirati/canvas-panel-web-components@latest"></script>
+        <script type="module" src="/_assets/javascript/lit/lightbox.js"></script>
+        <script type="module" src="/_assets/javascript/lit/modal.js"></script>
```

Comment:
Need to figure out a cleaner way to loop through all `lit-element` components and add them to `<head>`